src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,91 @@
+package duke;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+public class TaskList implements Serializable {
+    private ArrayList<Task> tasks;
+    private int numTask;
```

Comment:
Should this variable name be changed to a more descriptive one, since readers may confuse this with the task index instead? One suggested change is
```suggestion
    private int numberOfTasks;
```

Reply:
amended as of 734dcb9b1359b25907ac79f9f742e0c796f93a03